sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/RntbdTransportClient.java

```diff
@@ -456,7 +456,7 @@ private Options(final Builder builder) {
             this.maxRequestsPerChannel = builder.maxRequestsPerChannel;
             this.maxConcurrentRequestsPerEndpointOverride = builder.maxConcurrentRequestsPerEndpointOverride;
             this.receiveHangDetectionTime = builder.receiveHangDetectionTime;
-            this.requestTimeout = builder.requestTimeout;
+            this.tcpNetworkRequestTimeout = builder.tcpNetworkRequestTimeout;
```

Comment:
RNTBD has its own set of configuration which can be passed as json or java properties.

is `tcpNetworkRequestTimeout` also supported through the old rntbd config as well?
if we do please make sure we do proper conflict resolution (meaning the config provided from programmatic API should always override the rntbd config from the json properties format)

Reply:
Yes, RNTBD config usually can be override by system property or java properties, but not for the ones with public API.

So for example, since networkRequestTimeout is public, even though there is a override by system property or java properties, internally rntbd will always use what networkRequestTimeout programmatically configured (either the default value or customized by customer)